src/settings/ActionProfiles/ViewActionProfile.js

```diff
@@ -175,6 +157,18 @@ export class ViewActionProfile extends Component {
       </AppIcon>
     );
 
+    const header = renderProps => (
```

Comment:
Move this function to the class methods and rename it to "renderPaneHeader" for example, please.

Reply:
Done!